cmd/license_list.go

```diff
@@ -242,16 +242,29 @@ func DisplayLicenseListJson(bom *schema.BOM, writer io.Writer, flags utils.Licen
 	defer getLogger().Exit()
 
 	var licenseInfo schema.LicenseInfo
-	var lc []schema.CDXLicenseChoice
+	var lc []map[string]string
+
+	var line, keys []string
+	keys, _ = prepareReportTitleData(LICENSE_LIST_ROW_DATA, flags.Summary)
 
 	for _, licenseName := range bom.LicenseMap.KeySet() {
 		arrLicenseInfo, _ := bom.LicenseMap.Get(licenseName)
 
 		for _, iInfo := range arrLicenseInfo {
 			licenseInfo = iInfo.(schema.LicenseInfo)
-			if licenseInfo.LicenseChoiceTypeValue != schema.LC_TYPE_INVALID {
-				lc = append(lc, licenseInfo.LicenseChoice)
+			line, err = prepareReportLineData(
```

Comment:
In alignment with my statements regarding the JSON produced match the CycloneDX schema, I believe the correct approach that achieves what you like (and set a precedent for other APIs as well) would be to conditionally generate the non-standard JSON output only if the `--summary` flag was also provided in addition the `--format json` flag and value.